controllers/apply/lib/validate-model.js

```diff
@@ -99,7 +99,9 @@ module.exports = function validateModel(formModel) {
             .required()
     });
 
-    const validationResult = Joi.validate(formModel, formSchema);
+    const validationResult = Joi.validate(formModel, formSchema, {
+        allowUnknown: true
```

Comment:
I've added this because the new field classes have methods and class properties on them that it doesn't make sense to include in this validation. The class field types have their own validation so we can slowly move away from needing this rough schema validation, we don't use it at all on the funding proposal form.